recipes-hulks/aliveness/aliveness_git.bb

```diff
@@ -0,0 +1,190 @@
+SUMMARY = "aliveness"
+HOMEPAGE = "https://hulks.de"
+LICENSE = "GPL-3.0-only"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git/COPYING;md5=5b4473596678d62d9d83096273422c8c"
+
+inherit cargo pkgconfig
```

Comment:
Remove `pkgconfig`?

Reply:
I do not know why, but `hula` needs that. Thus, this was a copy paste... I removed it